examples/widgets/text_cursor.py

```diff
@@ -0,0 +1,313 @@
+"""
+==========
+Textcursor
```

Comment:
```suggestion
Annotated cursor
```
or similar.

Textcursor is often associtated with a cursor for text selection:
![grafik](https://user-images.githubusercontent.com/2836374/115466120-e8555080-a22f-11eb-9eec-98da271a81e6.png)

Reply:
Renamed .py file and the class itself accordingly.